### PR TITLE
Now use assets manifest

### PR DESCRIPTION
**Pull request by bricesanchez**
This is needed for Rails 6

### CODE REVIEW

**Comment by parndt** [comment]
Will it work for versions before 6?

**Comment by parndt** [comment]
Maybe this will help us with #53?

**Comment by bricesanchez** [comment]
@parndt At this point, This PR's CI runs against Rails 5.2.3.

**Comment by evenreven** [comment]
The new code in this PR doesn't seem to include all language files (only current locale got precompiled when I did a fresh deploy with an empty assets directory), and seems like it breaks bilingual admin setups? This worked in version 2.1 (due to the fix in my PR and due to the fact that all language js files got precompiled). In my case, `nb` language is precompiled because that's my default language (I guess? I'm no Sprockets expert), so it will mostly work for me. But once I try to change backend language (i.e. with `?set_locale=en`), the buttons get curly braces on them. (I don't do this often, but it should be possible, no?)

Screenshot_2019-10-19 Kunsthøgskolen i Oslo - Refinery

**Comment by parndt** [comment]
Thanks for letting us know @evenreven ! That's a shame.

File: lib/refinery/wymeditor/engine.rb [hunk 1]
```diff
@@ -8,14 +8,7 @@ class Engine < ::Rails::Engine
 
       # set the manifests and assets to be precompiled
       config.to_prepare do
-        Rails.application.config.assets.precompile += %w(
-          wymeditor.css
-          theme.css
-          wymeditor/lang/*
```
Comment (parndt) [inline lib/refinery/wymeditor/engine.rb]: @bricesanchez did we need this in the manifest?

**Comment by evenreven** [comment]
Did you find out about this? The fact that it stopped working once the wymeditor/lang/* line disappeared makes me think it needs to be in the manifest. I'm no Sprockets 4 expert, though, so I could be wrong (I've more or less ditched it in favour of Webpacker in my own app now).

**Comment by bricesanchez** [comment]
Hi @evenreven, i will have to test it, i've probably missed something

**Comment by evenreven** [comment]
@bricesanchez We did a bundle update because of another upgrade now, and this bug came back to bite me in the arse since I was suddenly on 2.2.0. It's not just a silent error either, a colleague who didn't know about my post here and assumed 2.1.0 and 2.2.0 were functionally identical found the bug via a sprockets exception, and did a local workaround in our assets initializer to solve it. This was on a clean install as well, no funny business in compiled assets or anything. Did you test this before merging? I don't mean to sound crass, it's just... [I spent a lot of time fixing and testing this five years old bug last September](https://github.com/parndt/refinerycms-wymeditor/pull/54) and within three weeks it was broken again. My fix also showed that these asset files were indeed necessary for bilingual setups to work. I understand that you don't necessarily want to ship dozens of (mostly unused) language files, but your i18n code seems to need them to work. 

If you want to not support bilingual admin setups, then that's fine (I guess), but at least say so. The issue was open for years before I fixed it, so I supposed it was a wanted addition.

**Comment by bricesanchez** [comment]
@evenreven I'm sorry i did not take the time to re-work on this bug. 
I'm in favor to support bilingual admin setups!

I think we could add all locales like this:

````
# app/assets/config/refinery_wymeditor_manifest.js

[...]
//= link wymeditor/lang/bg.js
//= link wymeditor/lang/ca.js
[...]
```

Or it could be great if we could use ERB in the manifest:

````
# app/assets/config/refinery_wymeditor_manifest.js.erb

[...]
<% Refinery::I18n.locales.each do |locale, _| %>
//= link wymeditor/lang/<% locale %>.js
<% end %>
[...]
```

What do you think @parndt?

**Comment by parndt** [comment]
@bricesanchez yep I'm happy to include all locales

**Pull request merged**